packages/frontend-shared/src/assets/browserLogos.ts

```diff
@@ -16,11 +17,12 @@ import genericBrowserLogo from '@packages/frontend-shared/src/assets/logos/gener
 export const allBrowsersIcons = {
   'Electron': electronIcon,
   'Chrome': chromeIcon,
+  'Chrome Beta': chromeBetaIcon,
+  'Chrome Canary': canaryIcon,
+  'Chrome Test': chromeTestIcon,
```

Comment:
@mschile What’s the purpose behind calling is ‘Chrome Test’ instead of ‘Chrome for Testing’?

Reply:
@mschile It seems strange to not call the browser by the name Chrome gave the browser, especially since it's the name reflected in the name of the browser used for other interactions with Cypress. 

Firefox Developer Edition similarly doesn't fit, and I would say the UI requires updates to accommodate these rather than changing the browser's name.

![Screenshot 2024-12-12 at 12 56 48 PM](https://github.com/user-attachments/assets/106f9b48-a9a1-48d9-aa5f-4e5424213a97)
![Screenshot 2024-12-12 at 12 57 20 PM](https://github.com/user-attachments/assets/b8771507-c4c9-44bd-af74-e6bdb0a44452)

Reply:
I'm good with updating the display name to `Chrome for Testing`.

Reply:
updates display name: [71576ec](https://github.com/cypress-io/cypress/pull/30751/commits/71576ecfb45cd7ae558e1e1114f30c25de141a41)